unitTests/utils/SearchboxResizeTest.ts

```diff
@@ -0,0 +1,87 @@
+import * as Mock from '../MockEnvironment';
+import { Searchbox } from '../../src/ui/Searchbox/Searchbox';
+import { ISearchboxOptions } from '../../src/ui/Searchbox/Searchbox';
+import { $$ } from '../../src/utils/Dom';
+
+interface HTMLElements {
+  CoveoOmniboxOrQuery: HTMLElement;
+  CoveoSearchButton: HTMLElement;
+  input: HTMLElement;
+  magicBoxIcon: HTMLElement;
+  magicBoxClear: HTMLElement;
+  magicBoxInput: HTMLElement;
+}
+
+function findHtmlElement(element: HTMLElement): HTMLElements {
+  return {
+    CoveoOmniboxOrQuery: $$(element).find('.CoveoOmnibox') || $$(element).find('.CoveoQuerybox'),
+    CoveoSearchButton: $$(element).find('.CoveoSearchButton'),
+    input: $$($$(element).find('.magic-box-input')).find('input'),
+    magicBoxIcon: $$(element).find('.magic-box-icon'),
+    magicBoxClear: $$(element).find('.magic-box-clear'),
+    magicBoxInput: $$(element).find('.magic-box-input')
+  };
+}
+
+export function SearchboxResizeTest() {
+  describe('SearchboxResize', () => {
+    var test: Mock.IBasicComponentSetup<Searchbox>;
+    describe('with an Omnibox as the Searchbox', () => {
+      it('should not be resize', () => {
+        test = Mock.optionsComponentSetup<Searchbox, ISearchboxOptions>(Searchbox, {
+          enableOmnibox: true
+        });
+        var structHTMLElement: HTMLElements = findHtmlElement(test.cmp.element);
+        for (let element in structHTMLElement) {
```

Comment:
Use a forEach loop so that it's less verbose.

This applies for all those tests (so that I'm not repeating myself too much) : 

```
getTestTargets(test.cmp.element).forEach(target=> expect($$(target).css('style')).toBe(''))
````

Reply:
I can't iterate through the propriety of an interface
https://stackoverflow.com/questions/45670705/typescript-iterate-interface-properties
I could do something like on the post of StackOverflow but i would be really similar